lib/graphql/adapters/base_job.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "active_job"
+
+module GraphQL
+  module Adapters
+    class BaseJob < ActiveJob::Base
+      DEFAULT_QUEUE_NAME = :default
+
+      queue_as { GraphQL::AnyCable.config.async_broadcasting["queue"] || DEFAULT_QUEUE_NAME }
+
+      def perform(serialized_payload, execute_method, event_name, args = {}, object = nil, options = {})
+        schema = schema_parse(serialized_payload)
+
+        schema.public_send(execute_method, event_name, args, object, **options)
+      end
+
+      private
+
+      def schema_parse(serialized_payload)
+        payload = JSON.parse(serialized_payload)
```

Comment:
Maybe, we can use symbolize_names here and avoid `#transform_keys`?

```suggestion
        payload = JSON.parse(serialized_payload, symbolize_names: true)
```

Reply:
Of course, yeah. We can just put symbolized hash to `ActiveJob` arguments, and it will work without `JSON.parse`